content/asp.net/getting-started/controllers-actions.md

```diff
@@ -1,20 +1,122 @@
+| /Person/Index/6 | Person     | Index   | The `Index` action on the `Person` controller will be invoked and the parameter `id`, if present, will be set to 6.                         |
+
+## Attribute routes
+When the globally-configured routes do not suit your needs you can override the defaults on a per-controller or per-action basis using routing attributes. For instance, you could decorate your `PersonController` with the following attribute to change the route from `/person` to `/peeps`:
+
+```c#
+[Route("peeps")]
+public class PersonController : Controller
+{
+    // ...
+}
+```
+
+Likewise, an action can be setup to use a different route if you apply an action to it, for instance, with the above `peeps` route set on the controller, you can have `peeps/favs` as the route for the action below with the following route applied:
+
+```c#
+[Route("peeps")]
+public class PersonController : Controller
+{
+    [Route("favs")]
+    public IActionResult Favorites()
+    {
+        //...
+    }
+}
+```
+
+In the above scenario, the action assumes the controller's route and builds upon it. You can also override the base for that specific action if you prepend the route with a forward slash:
+
+```c#
+    [Route("peeps")]
+    public class PersonController : Controller
+    {
+        [Route("favs")]
+        public IActionResult Favorites()
+        {
+            //...
+        }
+
+        [Route("/people/home")]
+        public IActionResult Index()
+        {
+            //...
+        }
+    }
+```
+
+Routes can get tricky to debug, so use them judiciously and know how they affect incoming requests in your app.  For example, someone looking at the above controller might make assumptions about the `Index` action, given the default route confiruration on the project; however, with the attributes applied as above, a request to `/person/index` would give them a `404 Not Found`.  Read more about routes [here](routing.md). 
+
+## Controller Helpers and Result Objects 
+In addition to the actions that you write yourself, you're also privy to the host of helper methods that are built into the base controller that you inherit from. These methods help to formulate a repsonse after you've completed any validation, service access or processing required. We've already looked at one as part of the samples above, but let's have another quick look:
+
+```c#
+public IActionResult Index()
+{
+    return View();
+}
+```        
+
+The call to the helper method `View` above gives you an easy way to say, "I'm done processing here, and now I would like the user to see the page related to this action."  The framework takes over, maintains the state of any properties you've set on the controller and renders a view following the default naming and location conventions we covered earlier.
+
+Just like `View`, there are other helpers that will make the lifting easier on the controller.
```

Comment:
"make the lifting easier" isn't a phrase I'm familiar with. I've heard "make life easier for" but even that doesn't sound right. How about "there are other helpers that will assist you in returning responses from your actions"